core/webhooks/chat.go

```diff
@@ -0,0 +1,48 @@
+package webhooks
+
+import (
+	"github.com/owncast/owncast/models"
+)
+
+// TODO: relocate to shared location that we and chat package can use
+const (
+	CHAT             = "CHAT"
+	NAMECHANGE       = "NAME_CHANGE"
+	PING             = "PING"
+	PONG             = "PONG"
+	VISIBILITYUPDATE = "VISIBILITY-UPDATE"
+)
```

Comment:
Trying to figure out where I can put this so it can be used by both the chat package and the webhook package.  Can refer to the one in the chat package or it ends up in a circular dependency issue

Reply:
Should we move it into `models` as an exported `ChatEvent` or something, and let all the other packages just pull it from models?

Reply:
I could.  Maybe a type similar to the Webhook Event Type?

Reply:
Yeah!  That should be an easy win against the circular dependency.